### PR TITLE
Update for index.md

### PR DESCRIPTION
**Pull request by alwinnocom**
Helpful info for new developers.

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -21,4 +28,13 @@ If you would like to make contributions to this documentation site, it's hosted
 
 * Talk to current developers in Discord.
 * Check [the list of issues](https://github.com/space-wizards/space-station-14/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for anything you can work on.
-* Reading through [Acronyms & Common Nomenclature](en/general-development/codebase-info/acronyms-and-nomenclature.md) as well as the other articles on this wiki in `Tips` or `Setup` will likely help you understand what is happening.
\ No newline at end of file
+* Reading through [Acronyms & Common Nomenclature](en/general-development/codebase-info/acronyms-and-nomenclature.md) as well as the other articles on this wiki in `Tips` or `Setup` will likely help you understand what is happening.
+
+### What repositories are part of Space Station 14?
+* [Space Station 14](https://github.com/space-wizards/space-station-14/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) is the main codebase. It has the most priority issues.
```
Comment (metalgearsloth) [inline src/index.md:34]: This includes your search query.

File: src/index.md [hunk 2]
```diff
@@ -21,4 +28,13 @@ If you would like to make contributions to this documentation site, it's hosted
...
+* Reading through [Acronyms & Common Nomenclature](en/general-development/codebase-info/acronyms-and-nomenclature.md) as well as the other articles on this wiki in `Tips` or `Setup` will likely help you understand what is happening.
+
+### What repositories are part of Space Station 14?
+* [Space Station 14](https://github.com/space-wizards/space-station-14/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) is the main codebase. It has the most priority issues. 
+* [Robust Toolbox](https://github.com/space-wizards/RobustToolbox/issues) is the custom game engine.
+
+There are other repositories that do not have as many priority issues to work on. If you are still curious about them, they are:
+- [Network](https://github.com/space-wizards/SpaceWizards.Lidgren.Network/issues)
+- [Server Launcher](https://github.com/space-wizards/SS14.Launcher/issues) 
+- [Other](https://github.com/space-wizards)
```
Comment (metalgearsloth) [inline src/index.md:40]: There's significantly more than this (e.g. RSIEdit, Arch) so better to just say there's others under space-wizards

File: src/index.md [hunk 3]
```diff
@@ -4,6 +4,13 @@
 This is the developer's wiki. If you are a player, you are looking for [the player wiki](https://wiki.spacestation14.io/wiki/Main_Page).
 ```
 
+```admonish question "Technical Issues"
+If you are having difficulties with logging into the game or building it, please try the following resources:
+[🔊 Discord](https://discord.com/invite/rGvu9hKffJ). This is the most active community of developers for Space Station 14. Ask a question on the General or Help channel.
```
Comment (metalgearsloth) [inline src/index.md:9]: Use the permanent link not this, https://discord.gg/ss14

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: src/index.md [hunk 4]
```diff
@@ -21,4 +28,11 @@ If you would like to make contributions to this documentation site, it's hosted
 
 * Talk to current developers in Discord.
 * Check [the list of issues](https://github.com/space-wizards/space-station-14/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for anything you can work on.
-* Reading through [Acronyms & Common Nomenclature](en/general-development/codebase-info/acronyms-and-nomenclature.md) as well as the other articles on this wiki in `Tips` or `Setup` will likely help you understand what is happening.
\ No newline at end of file
+* Reading through [Acronyms & Common Nomenclature](en/general-development/codebase-info/acronyms-and-nomenclature.md) as well as the other articles on this wiki in `Tips` or `Setup` will likely help you understand what is happening.
+
+### What repositories are part of Space Station 14?
+* [Space Station 14](https://github.com/space-wizards/space-station-14) is the main codebase. It has the most priority issues. [Space Station 14 Issues](https://github.com/space-wizards/space-station-14/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)
```
Comment (PJB3005) [inline src/index.md:34]: Still search query
```suggestion
* [Space Station 14](https://github.com/space-wizards/space-station-14) is the main codebase. It has the most priority issues. [Space Station 14 Issues](https://github.com/space-wizards/space-station-14/issues)
```

**Pull request merged**